hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -261,7 +261,7 @@ message OMResponse {
 
   optional ListTrashResponse                  listTrashResponse            = 91;
   optional RecoverTrashResponse               RecoverTrashResponse         = 92;
-  optional PurgePathsResponse                 purgePathsResponse           = 93;
+  optional PurgeDirectoriesRequest            purgeDirectoriesResponse     = 93;
```

Comment:
s/PurgeDirectoriesRequest/purgeDirectoriesResponse ??

Reply:
Thanks for catching this.